main.py

```diff
@@ -123,34 +120,33 @@ def createDataDict(checkledLink):
     pdf_link = f"https://dergipark.org.tr{shortLink}"
     dataDict['Yayın PDF'] = pdf_link
     
-    # Parsing the data has been done
-    # I'm adding the data to the data_list
+    # I'm adding Parsed data to the data_list
     data_list.append(dataDict)
 
-    print(f"{counter}. Article created [{checkCounter}. Article]")
+    print(f"{counter}. Article created")
 
+    # OUTPUT TO TXT FILE
```

Comment:
This comment section is unnecessarily long. Need to update it to make it shorter.